<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_Video">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>790</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox 
{
	border: 1px solid black;
}</string>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seek_backward">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_play_pause">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>480</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stop">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_seek_forward">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>480</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_mute">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>480</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_volume">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>500</y>
      <width>160</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_progress">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>631</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_total_time">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>460</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSelect_Video_file">
    <property name="title">
     <string>Select Video File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuSelect_Video_file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
